dockerfiles/cpubuilder/build_manylinux_x86_64.Dockerfile

```diff
@@ -50,11 +50,19 @@ RUN ./install_awscli.sh && rm -rf /install-awscli
 RUN yum install -y epel-release && \
     yum install -y gcc-toolset-12-libatomic-devel && \
     yum install -y patchelf && \
-    yum install -y gtest-devel && \
     yum install -y vim-common git-lfs && \
     yum clean all && \
     rm -rf /var/cache/yum
 
+######## Installing google test #######
+RUN git clone https://github.com/google/googletest.git && \
```

Comment:
Can You break this out to a shell script like the others above. Then in the shell script, delete the repo you checked out before exiting so it doesn't clog up the image.